webui/src/pages/auth/reset-password.jsx

```diff
@@ -0,0 +1,188 @@
+import React, {useRef, useState} from "react";
+import Layout from "../../lib/components/layout";
+import Row from "react-bootstrap/Row";
+import Card from "react-bootstrap/Card";
+import Form from "react-bootstrap/Form";
+import Col from "react-bootstrap/Col";
+import Button from "react-bootstrap/Button";
+import {
+    AlertIcon,
+    MailIcon
+} from "@primer/octicons-react";
+import {Error} from "../../lib/components/controls";
+import {auth} from "../../lib/api";
+import validator from "validator/es";
+
+const TOKEN = "token";
+
+const RequestResetPasswordForm = () => {
+
+    const onEmailChange = () => {
+        if (validator.isEmail(email.current.value)) {
+            setFormValid(true);
+            setEmailValid(true);
+        } else {
+            setFormValid(false);
+            setEmailValid(false);
+        }
+    };
+
+    const [reqResetPwdError, setReqResetPwdError] = React.useState(null);
```

Comment:
What is the difference between React.useState and useState?

Reply:
no idea, removed the React prefix thanks